kubectl-rook-ceph.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright © 2023 kubectl-rook-ceph
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import "github.com/kubectl-rook-ceph/cmd"
+
+func main() {
```

Comment:
Do we need this in the root dir, or can we follow the pattern of the rook repo, where there are no go files in the root? The main go file could be in the cmd dir.

Reply:
I'll check on this how we can do that.

Reply:
done